packages/block-editor/src/components/block-lock/menu-item.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useReducer } from '@wordpress/element';
+import { MenuItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import BlockLockModal from './modal';
+
+export default function BlockLockMenuItem( {
+	blockTitle,
+	clientId,
+	isLocked,
+} ) {
+	const [ isModalOpen, toggleModal ] = useReducer(
+		( isActive ) => ! isActive,
+		false
+	);
+
+	const label = isLocked
+		? sprintf(
+				/* translators: %s: block name */
+				__( 'Unlock %s' ),
+				blockTitle
+		  )
+		: sprintf(
+				/* translators: %s: block name */
+				__( 'Lock %s' ),
+				blockTitle
+		  );
+
+	return (
+		<>
+			<MenuItem onClick={ toggleModal }>{ label }</MenuItem>
```

Comment:
It might be good to try showing the lock icon here.

Reply:
Like others mentioned, lock/unlock icons might need an update.

![CleanShot 2022-03-04 at 14 39 54](https://user-images.githubusercontent.com/240569/156748661-79a86d89-6078-4507-ba1c-818fddc856a4.png)